docs/source/en/quicktour.mdx

```diff
@@ -115,23 +115,23 @@ Create a [`pipeline`] with the task you want to solve for and the model you want
 >>> speech_recognizer = pipeline("automatic-speech-recognition", model="facebook/wav2vec2-base-960h")
 ```
 
-Next, load a dataset (see the 🤗 Datasets [Quick Start](https://huggingface.co/docs/datasets/quickstart.html) for more details) you'd like to iterate over. For example, let's load the [SUPERB](https://huggingface.co/datasets/superb) dataset:
+Next, load a dataset (see the 🤗 Datasets [Quick Start](https://huggingface.co/docs/datasets/quickstart.html) for more details) you'd like to iterate over. For example, let's load the [MInDS-14](https://huggingface.co/datasets/PolyAI/minds14) dataset:
```

Comment:
@sgugger @stevhliu - is this ok for you? 

I would like to replace `superb` here with `PolyAI/minds14` because Superb has 6GB of data which takes usually around 5 minutes at 10MB/s download speed to preprocess and download. `PolyAI/minds14` is just ~400MB so it should only take around 40 seconds which makes the quicktour much nicer to follow IMO. 

Ok for you? 

Also nicer for the doc tests but that shouldn't be the main reason

Reply:
Completely fine by me!